app/frontend/components/pages/sendAda/sendAdaPage.tsx

```diff
@@ -160,10 +165,12 @@ const SendAdaPage = ({
     type: AssetFamily.ADA,
     policyId: null,
     assetName: 'ADA',
-    assetNameHex: null,
     fingerprint: null,
     quantity: balance,
-    star: true,
+    searchable: {
```

Comment:
specifying explicitly the searchable attributes to a dropdown item seems to me like coupling higher (search) and lower (token representation in dropdown) concerns, so I'd consider keeping these keys at the same level as the rest of keys and let the search logic decide which keys to filter by - what do you think?